configure.ac

```diff
@@ -349,22 +356,22 @@ AS_IF([   test $have_userns = yes],
 
 # image builders
 
-AS_IF([   test -n "$BUILDAH_VERSION"],
+AS_IF([   test "$BUILDAH_VERSION_NOTE" != "not found" ],
```

Comment:
Change these tests to e.g. `-n "$BUILDAH"`. Because `$BUILDAH_VERSION_NOTE` etc. is a human-readable message, we shouldn't use it in conditionals.